web-local/test/ui/dashboards.spec.ts

```diff
@@ -112,6 +117,28 @@ describe("dashboards", () => {
     // Check that the total records are 275 and have comparisons
     await playwrightExpect(page.getByText("275 -12 -4%")).toBeVisible();
 
+    // Check the row viewer accordion is updated
+    await playwrightExpect(
+      page.getByText("Source Data 275 of 100k rows")
+    ).toBeVisible();
+
+    // Check row viewer is collapsed
+    await playwrightExpect(
+      page.getByRole("button", { name: "7029" })
```

Comment:
Nit: I'm not too sure what "7029" is. Is that a data point in a cell?

Reply:
Done